scalyr_agent/monitor_utils/k8s.py

```diff
@@ -103,6 +104,13 @@
         "list": Template("/apis/apps/v1/namespaces/${namespace}/statefulsets"),
         "list-all": "/apis/apps/v1/statefulsets",
     },
+    "Rollout": {
```

Comment:
Might be good to explicitly point out in a comment that this is for ArgoCD enabled clusters, I'll leave it up to you if you think it's worth adding

Reply:
Added requested comments.